packages/app-accounts/src/translate.ts

```diff
@@ -4,5 +4,4 @@
 
 import { withNamespaces } from 'react-i18next';
 
-// @ts-ignore definition incorrect
```

Comment:
Yay, previous issue with typings in version addressed. (... and replace by a worse situation. Next time lucky.)